proposals/0255-Enhance-BodyInformation-vehicle-data.md

```diff
@@ -127,24 +171,27 @@ Following vehicle data params are deprecated from `BodyInformation` struct:
 	<param name="ignitionStatus" type="Common.IgnitionStatus" mandatory="true">
 		<description>Provides information on current ignitiion status. See IgnitionStatus.</description>
 	</param>
-	<param name="driverDoorAjar" type="Boolean" mandatory="false" deprecated="true" since="X.x">
+	<param name="driverDoorAjar" type="Boolean" mandatory="false">
```

Comment:
Is removing ` deprecated="true" since="X.x">` intentional?  If so, please include rationale for change in PR description.

Reply:
This was an error in accepted proposal as HMI API does not support deprecated/since etc. I have maintained this information in MOBILE API and in description for these deprecated params in HMI API.